src/Jackett.Common/Indexers/LostFilm.cs

```diff
@@ -707,27 +712,35 @@ private async Task<List<ReleaseInfo>> FollowTrackerRedirection(string url, Track
                         var titleComponents = new string[] {
                             serieTitle, details.GetEpisodeString(), episodeName, techInfo
                         };
-                        release.Title = string.Join(" - ", titleComponents.Where(s => !string.IsNullOrEmpty(s)));
-
                         var downloadLink = row.QuerySelector("div.inner-box--link > a");
-                        release.Link = new Uri(downloadLink.GetAttribute("href"));
-                        release.Guid = release.Link;
-
                         var sizeString = releaseDetails.Groups["size"].Value.ToUpper();
                         sizeString = sizeString.Replace("ТБ", "TB"); // untested
                         sizeString = sizeString.Replace("ГБ", "GB");
                         sizeString = sizeString.Replace("МБ", "MB");
                         sizeString = sizeString.Replace("КБ", "KB"); // untested
-                        release.Size = ReleaseInfo.GetBytes(sizeString);
-
-                        // add missing torznab fields not available from results
-                        release.Seeders = 1;
-                        release.Peers = 2;
-                        release.DownloadVolumeFactor = 0;
-                        release.UploadVolumeFactor = 1;
-                        release.MinimumRatio = 1;
-                        release.MinimumSeedTime = 172800; // 48 hours
+                        var guid = new Uri(downloadLink.GetAttribute("href"));
```

Comment:
The correct var name is link or downloadLink. guid means unique id and can be anything, in other trackers we use the comment link, the magnet link.